test/integration/default/snap_test.rb

```diff
@@ -0,0 +1,33 @@
+describe package('solarwinds-snap-agent') do
+  it { should be_installed }
+end
+
+describe service('swisnapd') do
+  it { should be_enabled }
+  it { should be_running }
+end
+
+describe file('/opt/SolarWinds/Snap/etc/config.yaml') do
+  it { should exist }
+  it { should be_owned_by 'solarwinds' }
+  its('group') { should eq 'solarwinds' }
+  its('mode') { should cmp '0640' }
+  its('content') { should match(%r{test-tags: test-kitchen}) }
+end
+
+describe file('/opt/SolarWinds/Snap/etc/plugins.d/publisher-processes.yaml') do
+  it { should exist }
+  it { should be_owned_by 'solarwinds' }
+  its('group') { should eq 'solarwinds' }
+  its('mode') { should cmp '0640' }
+  its('content') { should match(%r{hostname_alias: test-kitchen}) }
+end
+
+describe file('/opt/SolarWinds/Snap/etc/plugins.d/publisher-appoptics.yaml') do
```

Comment:
please add checks for default tasks (tasks-autoload.d/)

Reply:
Added